test/optimizers/test_spsa.py

```diff
@@ -74,13 +74,13 @@ def objective(x):
             spsa = SPSA(**settings)
 
         with self.assertWarns(DeprecationWarning):
-            result = spsa.optimize(circuit.num_parameters, objective, initial_point=initial_point)
+            result = spsa.minimize(objective, x0=initial_point)
```

Comment:
A deprecation warning is raised from the evolved operator ansatz, I believe, because it still uses opflow in some methods. This also happens in some of the tests from #17, where the assertion is still neided.

Reply:
Ok, this more that we should be changing the code to avoid that deprecation if possible. Its not really something that we are raising in our code and we want to make sure it is and gets seen by the end-user. For this case that assertWarns should not be there - I can remove it (outdenting the call to minimize of course) and it passes fine.

Reply:
Since its passing CI lets go with this - it is likely that once the opflow/QI logic use here is all removed that this will need to be revisited.